guides/common/modules/proc_upgrading-hosts-to-next-major-release.adoc

```diff
@@ -36,9 +38,8 @@ For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{conte
 .. In the *Job Template* dropdown menu, select your template.
 .. Click *Submit*.
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Select the hosts that you want to upgrade to the next major {RHEL} version.
-. In the upper right of the Hosts window, from the *Select Action* list, select *Preupgrade check with Leapp*.
-. Enter the required information and start the pre-upgrade check.
+. Select the hosts that you want to upgrade to the next major {EL} version.
+. In the *Schedule Remote Job* drop down menu, select *Preupgrade check with Leapp*.
```

Comment:
```suggestion
. From the *Schedule Remote Job* list, select *Preupgrade check with Leapp*.
```
This is recommended by the IBM style guide, and IMHO shorter name for an UI element is better.

Reply:
Also, I see you removed "Enter the required information and start the pre-upgrade check.". While this seems like on obvious thing to do, if users still need to do something manually in there, I would rather be explicit about it. I am unable to verify right now, so if all the things concerning the check are automatic, you can ignore this comment.

Reply:
You cannot enter any information (at least I could not): the REX job just starts and you have to "search" for the URL on the same page under "Recent jobs"; it does not return any information or forward you somewhere. Can you reproduce this?

Reply:
I cannot, but I am going to trust your word. You can ignore my second comment in this thread.

Reply:
No rush with this PR; maybe someone else wants to have a look at my diff next week.

Reply:
Applied your initial suggestion from https://github.com/theforeman/foreman-documentation/pull/2778#discussion_r1484440638 :heavy_check_mark: